examples/csv_to_schemaorg.py

```diff
@@ -1,37 +1,49 @@
+#!/usr/bin/env python3
+
 import os
+import re
 import json
-
+import argparse
 import pandas as pd
 
 from schematic.schemas.explorer import SchemaExplorer
-
 from schematic.utils.csv_utils import create_schema_classes
-from schematic.utils.config_utils import load_yaml
+from schematic import CONFIG
 
-from definitions import CONFIG_PATH, DATA_PATH
+# Constants (to avoid magic numbers)
+FIRST = 0
 
-config_data = load_yaml(CONFIG_PATH)
+# Create command-line argument parser
+parser = argparse.ArgumentParser(allow_abbrev=False)
+parser.add_argument("schema_csv_list", nargs="+", metavar="CSV SCHEMA(S)", help="Input CSV schema file(s).")
+parser.add_argument("--output_jsonld", "-o", help="Output JSON-LD schema file.")
+parser.add_argument("--config", "-c", help="Configuration YAML file.")
+args = parser.parse_args()
 
-# path to base schema
-base_schema_path = os.path.join(DATA_PATH, '', config_data["model"]["biothings"]["location"])
-
-# schema name (used to name schema json-ld file as well)
-output_schema_name = "HTAN"
-
-# schema extension definition csv files
-schema_extensions_csv = [
-                        os.path.join(DATA_PATH, '', 'csv/HTAN.csv')
-                        ]
+# Load configuration
+config_data = CONFIG.load_config(args.config)
+base_schema_path = CONFIG["model"]["biothings"]["location"]
 
 # instantiate schema explorer
 base_se = SchemaExplorer()
 
 # load base schema (BioThings)
 base_se.load_schema(base_schema_path)
 
-for schema_extension_csv in schema_extensions_csv:
+for schema_extension_csv in args.schema_csv_list:
```

Comment:
@sujaypatil96 once this PR is reviewed, merged and tested,  we can think if
- this multi-csv functionality + 
- your schema csv parser that incrementally changes the schema graph 
can be used in the solution for #296